WordPress/src/test/java/org/wordpress/android/ui/mysite/MySiteViewModelTest.kt

```diff
@@ -2250,6 +2253,20 @@ class MySiteViewModelTest : BaseUnitTest() {
         assertThat((getDashboardTabLastItems()[0] as SingleActionCard).imageResource).isEqualTo(expected)
     }
 
+    @Test
+    fun `JP migration scuccess card click should be tracked`() {
```

Comment:
NP: typo

Reply:
Thank you for noticing this typo @mkevins 🙇 

Fixed in: https://github.com/wordpress-mobile/WordPress-Android/pull/17585/commits/7c508969ff62f337d67f44a62b9579ad929a45d8